resotocore/resotocore/task/scheduler.py

```diff
@@ -18,7 +18,7 @@ def __init__(self) -> None:
             # max_instances: allowed parallel instances for one job
             # misfire_grace_time: seconds after the designated runtime that the job is still allowed to be run
             job_defaults={"coalesce": True, "max_instances": 32, "misfire_grace_time": 3600},
-            timezone=utc,
+            timezone=ZoneInfo("Etc/UTC"),
```

Comment:
Do we want to use the local time zone here and fallback to UTC?

Reply:
yes!